internal/test/e2e/run.go

```diff
@@ -128,6 +128,7 @@ func (e *E2ESession) runTests(regex string) error {
 		e.uploadDiagnosticArchiveFromInstance(regex)
 		return fmt.Errorf("error running e2e tests on instance %s: %v", e.instanceId, err)
 	}
+	e.commandId = commandId
```

Comment:
Instead of storing it in the session I think the `commandID` can just be another return value from this method

Reply:
yeah that's fair